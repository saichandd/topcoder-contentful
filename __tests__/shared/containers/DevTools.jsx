/* TODO: This test is temporarly disabled. The problem is that this test cannot
 * work in prod until we find a way to mock a non-installed module. I believe,
 * Babel should be able to help here. */

/*
import _ from 'lodash';

test('placeholder', _.noop);
*/


import React from 'react';
import Rnd from 'react-test-renderer/shallow';

const DevTools = require('containers/DevTools').default;

const rnd = new Rnd();
// const store = createStore(() => ({}), {}, DevTools.instrument());

test('Snapshot match', () => {
  rnd.render(<DevTools />);
  expect(rnd.getRenderOutput()).toMatchSnapshot();
});
