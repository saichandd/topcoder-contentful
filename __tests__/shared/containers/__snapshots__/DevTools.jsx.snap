// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot match 1`] = `
<ReduxDevtoolsDockMonitor
  changePositionKey="ctrl-p"
  toggleVisibilityKey="ctrl-m"
>
  <ReduxDevtoolsLogMonitor
    theme="tomorrow"
  />
</ReduxDevtoolsDockMonitor>
`;
